### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by jumalaw98**
modified:   README.md

# Description

Updated the readme file for easy contribution steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,36 +7,60 @@ A frontend development for the SpaceYaTech Mastori blog application. This is a c
...
+## Set up the project 
 
-To get started with contributing to the blog frontend, follow these steps:
+### Prerequisites 
 
-1. Clone this repository to your local machine: https://github.com/SpaceyaTech/blog-frontend.git
+To run the project on to your system, need to have the following installed .
 
-2. Install dependencies: npm install or yarn install
+- [Node](https://nodejs.org/en/download/) and [NPM packages](https://www.npmjs.com/package/download),  all latest versions.
```
Comment (nyandika) [inline README.md]: npm comes bundled in node

**Review by nyandika — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jumalaw98) [reply README.md]: hea, @nyandika so on this I should only write the latest node version as required.

Comment (nyandika) [reply README.md]: Yea, you can remove the link to npm

**Review by nyandika — DISMISSED** [review]

**Review by JimmyTron — APPROVED** [review]
Fixed the merge conflicts.

**Review by nyandika — APPROVED** [review]
LGTM!

**Pull request merged**